insights/parsers/lsvmbus.py

```diff
@@ -0,0 +1,70 @@
+"""
+LsvmBus - Command ``lsvmbus -vv``
+=================================
+
+This module parses the output of the command ``lsvmbus -vv``.
+"""
+import re
+
+from insights import LogFileOutput, parser, CommandParser
+from insights.parsers import SkipException
+from insights.specs import Specs
+
+
+@parser(Specs.lsvmbus)
+class LsvmBus(CommandParser, LogFileOutput):
```

Comment:
I'm OK with the logic, but just curious why adding the `LogFileOutput` as a base?

Reply:
Hi @xiangce Actually `LogFileOutput` as particular significance here. It was left out from my side unintentionally and I forgot to mark this PR as **WIP** which caused confusion.

Now it is ready for review. Thanks.